src/Database/Monai.Deploy.InformaticsGateway.Database.csproj

```diff
@@ -70,5 +70,6 @@
     <PackageReference Include="AspNetCore.HealthChecks.MongoDb" Version="6.0.2" />
     <PackageReference Include="Microsoft.Extensions.Diagnostics.HealthChecks.EntityFrameworkCore" Version="6.0.21" />
     <PackageReference Include="Microsoft.Extensions.Options.ConfigurationExtensions" Version="6.0.0" />
+    <PackageReference Include="NLog.Web.AspNetCore" Version="5.3.3" />

```

Comment:
Is this needed since this project doens't reference any ASP.NET lib.

Reply:
not sure how that go add. will remove